gc/base/segregated/SweepSchemeSegregated.hpp

```diff
@@ -50,6 +50,7 @@ class MM_SweepSchemeSegregated : public MM_BaseVirtual
 protected:
 	MM_MemoryPoolSegregated *_memoryPool;
 	MM_MarkMap *_markMap;
+	MM_EnvironmentBase *_environment;
```

Comment:
can we use MM_GCExtensions instead? This structure has only one instance but can be used by multiple threads

Reply:
Done.